superset-frontend/src/components/IconButton/index.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { styled } from '@superset-ui/core';
+import Button from 'src/components/Button';
+import { ButtonProps as AntdButtonProps } from 'antd/lib/button';
+
+export interface IconButtonProps extends AntdButtonProps {
+  buttonText: string;
+  icon: string;
```

Comment:
Is it `icon` or `logo`?

BTW, I prefer `icon`, since it's more generic.

Reply:
It is now `icon`, I changed it this morning at [`this commit`](https://github.com/apache/superset/pull/14818/commits/f3f625d7e1d863aa5ea74d00492a6758fb9adcdb) as per [Elizabeth's request](https://github.com/apache/superset/pull/14818#discussion_r641049810).

Reply:
You need to update `IconButton.test.jsx` and the PR description then :)

Reply:
OOPS haha missed those, thank you for that catch! Fixed the description and the tests at [`this commit`](https://github.com/apache/superset/pull/14818/commits/c8b148e365ec614654f3c1eee30d932c85b051b0).